src/lib.rs

```diff
@@ -169,7 +169,7 @@ impl MasterNode {
 
         // get new proof
         let new_proof = get_proof(&self.rpc, *staking_authority);
-        log::info!("new challenge: {:?}", new_proof.challenge);
+        // log::info!("new challenge: {:?}", new_proof.challenge);
```

Comment:
why rm

Reply:
but why do u want to see the challenge. Maybe it should be with trace